### PR TITLE
Update process Ecosystem

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

This PR
- Removes Mock data for ecosystem.
- Replaces hard coded data with blocks for Tools, Contributors and Organisations.
- Updates Card components as per the design. [Individual Single page view designs will be updated in another PR](#478).
- Adds Icons for tool card.
- Renames people to contributors.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kelvinkipruto — APPROVED** [review]

File: apps/charterafrica/src/components/EcosystemList/useEcosystemList.js [hunk 1]
```diff
@@ -5,9 +5,13 @@ import fetchJson from "@/charterafrica/utils/fetchJson";
...
-  const { collection, ...rest } = params;
-  const searchParams = new URLSearchParams(rest).toString();
-  const key = `/api/v1/resources/collections/${collection}${
+  const param = new URLSearchParams();
+  Object.keys(params).forEach((key) => {
+    if (params[key] && params[key] !== "undefined")
+      param.append(key, params[key]);
+  });
+  const searchParams = param.toString();
+  const key = `/api/v1/resources/collections${
```
Comment (kilemensi) [inline apps/charterafrica/src/components/EcosystemList/useEcosystemList.js:14]: 1. See how `useDocuments`, `useDatasets`, `useArticles`, etc. have been written.
2. Isn't `useEntities` a better name or is this something completely different?

File: apps/charterafrica/src/components/ToolCard/ToolCard.js [hunk 2]
```diff
@@ -49,6 +61,7 @@ const ToolCard = React.forwardRef(function ToolCard(props, ref) {
                 width: "100%",
                 maxWidth: horizontalDisplay ? 585 : "100%",
                 minWidth: 330,
+                height: horizontalDisplay ? "100%" : 160,
```
Comment (kilemensi) [inline apps/charterafrica/src/components/ToolCard/ToolCard.js:64]: Is `orientation` like `direction`? I think sticking to `direction` like MUI layout components (`Stack`, `Grid`, etc.) may work best where `direction = [row|column]`

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

Haven't looked at page processing yet but looking at the Airtable [base](https://airtable.com/appDuwqg5qeS4h5ji/tblMDIuWx6bQO69lu/viwFMaFcSj3r3vsWp?blocks=hide), I don't see things like external id anymore. Is the Ecosystem global up to date with latest changes?

**Comment by koechkevin** [comment]
> 👍🏽
> 
> Haven't looked at page processing yet but looking at the Airtable [base](https://airtable.com/appDuwqg5qeS4h5ji/tblMDIuWx6bQO69lu/viwFMaFcSj3r3vsWp?blocks=hide), I don't see things like external id anymore. Is the Ecosystem global up to date with latest changes?

It will be mapped to code link.

**Comment by kilemensi** [comment]
> > 👍🏽
> > Haven't looked at page processing yet but looking at the Airtable [base](https://airtable.com/appDuwqg5qeS4h5ji/tblMDIuWx6bQO69lu/viwFMaFcSj3r3vsWp?blocks=hide), I don't see things like external id anymore. Is the Ecosystem global up to date with latest changes?
> 
> It will be mapped to code link.
>
Where then are we checking filtering links that point to GitHub only @koechkevin? e.g. what happens when code link is "https://gitlab.com/data-analysis5/dna-sequencing/54gene-wgs-germline"

File: apps/charterafrica/src/lib/ecosystem/collections.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {
```
Comment (kilemensi) [inline apps/charterafrica/src/lib/ecosystem/collections.js:1]: Why is this here? Is it useful anywhere else besides the /v1/apis/?

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: apps/charterafrica/src/lib/ecosystem/collections.js (hunk 3 above)
Comment (koechkevin) [reply apps/charterafrica/src/lib/ecosystem/collections.js:1]: @kilemensi at the moment this is only useful in ecosystem since the only processed collections are tools, organisations and contributors

File: apps/charterafrica/src/pages/api/v1/resources/[type].page.js [hunk 4]
```diff
@@ -1,8 +1,42 @@
...
 import fetchDatasets from "@/charterafrica/lib/openAfrica";
 import { fetchDocuments } from "@/charterafrica/lib/sourceAfrica";
 
+const collectionMap = {
+  organisations: getServerSideOrgs,
+  contributors: getServerSideContributors,
+  tools: getServerSideTools,
+};
+
+async function collections(req, res) {
```
Comment (kilemensi) [inline apps/charterafrica/src/pages/api/v1/resources/[type].page.js:17]:
```suggestion
const ecosystemMap = {
  organisations: getServerSideOrgs,
  contributors: getServerSideContributors,
  tools: getServerSideTools,
};

async function ecosystem(req, res) {
```

**Review by kilemensi — APPROVED** [review]
LGTM
